corehq/motech/repeaters/models.py

```diff
@@ -1327,3 +1330,10 @@ def domain_can_forward(domain):
         domain_has_privilege(domain, ZAPIER_INTEGRATION)
         or domain_has_privilege(domain, DATA_FORWARDING)
     )
+
+
+def domain_can_forward_now(domain):
```

Comment:
I'll add docstrings. We can't combine them because they are used differently. -- When `domain_can_forward()` is False, we don't register a repeat record.

Reply:
d1119bbec47